openqa-trigger-bisect-jobs

```diff
@@ -311,6 +311,8 @@ def main(args):
             )
         except subprocess.SubprocessError as err:
             if 'the repositories for the below updates are unavailable' in str(err.stderr):
+                extra_comment = "Bisects exits due to: "
```

Comment:
I don't understand the language here. What do you mean with "Bisects exits"?

Reply:
I wanted to say the no more bisects jobs. but I guess is better for you `openqa-clone-job` exits... right?

Reply:
Sorry, I also don't understand that language.